Profiles - ZIB 2017/nl-core-patient.xml

```diff
@@ -306,6 +310,25 @@
       <path value="Patient.extension.valueCodeableConcept.coding.code" />
       <min value="1" />
     </element>
+    <element id="Patient.extension:lifeStance">
+      <path value="Patient.extension" />
+      <sliceName value="lifeStance" />
+      <min value="0" />
+      <max value="*" />

```

Comment:
The zib defines a cardinality of max 1 for the chosen lifestance Although this is not an explicit constraint on the number of allowed lifestances per patient (since it's possible to bind multiple instances of the zib to a patient), I wonder if this shouldn't be interpreted as a max of 1.